tutorial/manipulate/index.md

```diff
@@ -116,21 +116,21 @@ CREATE TABLE "tmp_level" AS
 
 CREATE TABLE "tmp_opportunity" AS 
     SELECT *, CASE 
-        WHEN "Probability" < 50 THEN 'Poor' 
-        WHEN "Probability" < 70 THEN 'Good' 
+        WHEN "probability" < 50 THEN 'Poor' 
+        WHEN "probability" < 70 THEN 'Good' 
```

Comment:
patri to sem? mel jsem za to, ze jsi to chtel zase vratit zpatky?

Reply:
Ano, to je samozrejme blbost, nekde mi to ulitlo Revertuju